### ISSUE TITLE
Training pixart sigma with any aspect ratio?

### ISSUE THREAD

**Issue by NguyenNhoTrung** [PixArt-alpha/PixArt-sigma#83]
I have a question: How to train pixart sigma with any aspect ratio (for example: image 960x1280 pixel). Can you describe details in step by step? Thank you!

**Comment by lawrence-cj** [comment]
This part is done during the dataset sampling. You can find it here: https://github.com/PixArt-alpha/PixArt-sigma/blob/f999a89ce86d2dc2b65d01e77602a4d2f4ddec85/train_scripts/train.py#L431

**Comment by NguyenNhoTrung** [comment]
I see that: here is config for aspect ratio.

![image](https://github.com/PixArt-alpha/PixArt-sigma/assets/93875903/e0fb5db8-eae8-4bdc-b814-7906b613ed75)
Can I add any aspect ratio into config ? (example: add 0.625 aspect ratio with 720x1280).  
When adding like this, does it decrease performance ?

**Comment by NguyenNhoTrung** [comment]
> This part is done during the dataset sampling. You can find it here:
> 
> https://github.com/PixArt-alpha/PixArt-sigma/blob/f999a89ce86d2dc2b65d01e77602a4d2f4ddec85/train_scripts/train.py#L431

**Comment by lawrence-cj** [comment]
Of course, you can add any aspect ratio into the config. It'll not cause a decreasing in performance IMO.
